hack/load-jenkins-pipelines.sh

```diff
@@ -10,7 +10,7 @@ INFRA_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/.." && pwd -P)"
 
 if [[ -z "${WHAT}" ]]; then
     pushd ${INFRA_ROOT}
-    folders=(`find jobs/pipelines -name Jenkinsfile -print | grep -v powervm | sed "s|jobs/pipelines/||g" | sed "s|/Jenkinsfile||" | xargs -I"{}" dirname {} | uniq`)
+    folders=(`find jobs/pipelines -name Jenkinsfile -print | sed "s|jobs/pipelines/||g" | sed "s|/Jenkinsfile||" | xargs -I"{}" dirname {} | uniq`)
```

Comment:
we need to a few more changes , can you take a look at this https://github.com/sudeeshjohn/jenkins-infra/commit/5062dff2cbc591f75c494b2b01760cfff5c464c2